
from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_DB_menu(object):
    def setupUi(self, DB_menu):
        DB_menu.setObjectName("DB_menu")
        DB_menu.resize(800, 600)
        self.centralwidget = QtWidgets.QWidget(DB_menu)
        self.centralwidget.setObjectName("centralwidget")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setGeometry(QtCore.QRect(30, 20, 141, 21))
        font = QtGui.QFont()
        font.setFamily("Times New Roman")
        font.setPointSize(14)
        self.label.setFont(font)
        self.label.setObjectName("label")
        self.label_2 = QtWidgets.QLabel(self.centralwidget)
        self.label_2.setGeometry(QtCore.QRect(200, 20, 161, 16))
        font = QtGui.QFont()
        font.setFamily("Times New Roman")
        font.setPointSize(14)
        self.label_2.setFont(font)
        self.label_2.setObjectName("label_2")
        self.label_3 = QtWidgets.QLabel(self.centralwidget)
        self.label_3.setGeometry(QtCore.QRect(30, 80, 141, 16))
        font = QtGui.QFont()
        font.setFamily("Times New Roman")
        font.setPointSize(14)
        self.label_3.setFont(font)
        self.label_3.setObjectName("label_3")
        self.label_4 = QtWidgets.QLabel(self.centralwidget)
        self.label_4.setGeometry(QtCore.QRect(30, 120, 131, 21))
        font = QtGui.QFont()
        font.setFamily("Times New Roman")
        font.setPointSize(14)
        self.label_4.setFont(font)
        self.label_4.setObjectName("label_4")
        self.comboBox = QtWidgets.QComboBox(self.centralwidget)
        self.comboBox.setGeometry(QtCore.QRect(180, 80, 221, 22))
        font = QtGui.QFont()
        font.setFamily("Times New Roman")
        font.setPointSize(14)
        self.comboBox.setFont(font)
        self.comboBox.setObjectName("comboBox")
        self.comboBox_2 = QtWidgets.QComboBox(self.centralwidget)
        self.comboBox_2.setGeometry(QtCore.QRect(180, 120, 221, 22))
        font = QtGui.QFont()
        font.setFamily("Times New Roman")
        font.setPointSize(14)
        self.comboBox_2.setFont(font)
        self.comboBox_2.setObjectName("comboBox_2")
        self.pushButton = QtWidgets.QPushButton(self.centralwidget)
        self.pushButton.setGeometry(QtCore.QRect(30, 170, 191, 31))
        font = QtGui.QFont()
        font.setFamily("Times New Roman")
        font.setPointSize(14)
        self.pushButton.setFont(font)
        self.pushButton.setObjectName("pushButton")
        self.pushButton_2 = QtWidgets.QPushButton(self.centralwidget)
        self.pushButton_2.setGeometry(QtCore.QRect(240, 170, 191, 31))
        font = QtGui.QFont()
        font.setFamily("Times New Roman")
        font.setPointSize(14)
        self.pushButton_2.setFont(font)
        self.pushButton_2.setObjectName("pushButton_2")
        self.label_5 = QtWidgets.QLabel(self.centralwidget)
        self.label_5.setGeometry(QtCore.QRect(30, 50, 141, 21))
        font = QtGui.QFont()
        font.setFamily("Times New Roman")
        font.setPointSize(14)
        self.label_5.setFont(font)
        self.label_5.setObjectName("label_5")
        self.comboBox_3 = QtWidgets.QComboBox(self.centralwidget)
        self.comboBox_3.setGeometry(QtCore.QRect(180, 50, 221, 21))
        self.comboBox_3.setObjectName("comboBox_3")
        DB_menu.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(DB_menu)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 800, 21))
        self.menubar.setObjectName("menubar")
        DB_menu.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(DB_menu)
        self.statusbar.setObjectName("statusbar")
        DB_menu.setStatusBar(self.statusbar)

        self.retranslateUi(DB_menu)
        QtCore.QMetaObject.connectSlotsByName(DB_menu)

    def retranslateUi(self, DB_menu):
        _translate = QtCore.QCoreApplication.translate
        DB_menu.setWindowTitle(_translate("DB_menu", "DB"))
        self.label.setText(_translate("DB_menu", "Преподаватель:"))
        self.label_2.setText(_translate("DB_menu", "TextLabel"))
        self.label_3.setText(_translate("DB_menu", "Предмет:"))
        self.label_4.setText(_translate("DB_menu", "Вид занятия:"))
        self.pushButton.setText(_translate("DB_menu", "Назад"))
        self.pushButton_2.setText(_translate("DB_menu", "Сформровать отчет"))
        self.label_5.setText(_translate("DB_menu", "Группа:"))
